sklearn/ensemble/_voting.py

```diff
@@ -90,7 +90,36 @@ def fit(self, X, y, sample_weight=None):
             self.named_estimators_[name] = current_est
 
         return self
-
+    
+    def fit_transform(self, X, y=None, **fit_params):
+        """
+        Return class labels or probabilities for X for each estimator.
```

Comment:
The one-line summary is usually placed in the first line (after the ```"""```) and does not use variable names.

Reply:
hey, @alfaro96  thanks for your suggestions, I will edit PR.